.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
This file as well as the whole `.vs` file should be untracked and removed from the repo.